README.md

```diff
@@ -33,6 +33,7 @@ These videos on TikTok, YouTube and Instagram get MILLIONS of views across all p
 1. Clone this repository
 2. Rename `.env.template` to `.env` and replace all values with the appropriate fields. To get Reddit keys (**required**), visit [the Reddit Apps page.](https://www.reddit.com/prefs/apps) TL;DR set up an app that is a "script". Copy your keys into the `.env` files.
 3. Run `pip3 install -r requirements.txt`
+4. Run `playwright install`
```

Comment:
_I'm not familiar with Python so ignore this comment if it doesn't make sense_

Do we need to install this in order for the app to work? from looking at their page this is meant for testing, if this is optional then we should say that it is, and also explain what this does in a super short testing (ex: use this for testing)